MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "bazel_features",
+    version = "0.0.0",
```

Comment:
nit: `0.0.1`?

Reply:
I'm planning to use publish-to-bcr, which allows you to leave the version at 0.0.0 (and only patch it when we push a tag)

Reply:
SGTM